students/jungchan/users/views.py

```diff
@@ -33,4 +33,22 @@ def post(self, request):
             return JsonResponse({'message':'SUCCESS'}, status=201)
         except KeyError:
             return JsonResponse({'message':'KeyError'}, status=400) 
-            
\ No newline at end of file
+            
+class LoginView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            email = data['email']
+            password = data['password']
+
+            if not User.objects.filter(email=email).exists():
+                return JsonResponse({'message':'INVALID_EMAIL'}, status=401)
+
+            if not User.objects.filter(password=password).exists():
+                return JsonResponse({'message':'INVALID_PASSWORD'}, status=401)
+
+            if User.objects.filter(email=email, password=password).exists():
+                return JsonResponse({'message':'SUCCESS'}, status=200)
```

Comment:
- 프론트로부터 입력된 email과 password가 동시에 만족하는 유저가 존재하지 않을 경우만 error을 주면 될 것 같습니다.
- 따라서 위의 각 각 판단한 조건들은 불필요할 것 같아요.
- 보안상 email이 틀렸는지, 비밀번호가 틀렸는지를 안 나타내는 것이 좋습니다. 그래서 에러 메시지도 INVALIDE_USER로 수정해 주세요.

```suggestion
            if not User.objects.filter(email=email, password=password).exists():
                return JsonResponse({'message':'INVALID_USER'}, status=401)
            
            return JsonResponse({'message':'SUCCESS'}, status=200)
```